src/index.ts

```diff
@@ -11,27 +12,30 @@ const main = async () => {
   logger.settings.minLevel = argv.debug ? 2 : argv.ci ? 4 : 3;
 
   switch (argv._[0]) {
-    case "configure":
+    case "configure": {
       const { configure } = await import("./configure");
       await configure(argv);
       break;
-    case "review":
+    }
+    case "review": {
       const { review } = await import("./review");
       const { getReviewFiles } = await import("./common/utils/getReviewFiles");
       const files = await getReviewFiles(argv.ci, argv.remote);
       await review(argv, files, openAIApiKey);
       break;
-    case "test":
+    }
+    case "test": {
       const { test } = await import("./test");
       await test(argv, openAIApiKey);
       break;
+    }
```

Comment:
Do you need these brackets?

Reply:
The linter was complaining about them 🤔 I guess I can see an argument for them making it clearer where each case ends, but I can undo this if you dislike it

Reply:
I looked it up and this is the reason https://stackoverflow.com/questions/50752987/eslint-no-case-declaration-unexpected-lexical-declaration-in-case-block#:~:text=An%20easy%20fix%20is%20to,add%20break%20in%20some%20cases.